Add test.bundle to documented test configuration tags in junit upload

AdrianLC opened the pull request:
### What does this PR do?
Add `test.bundle` tag to the list of test configuration tags that users can specify manually in junit upload.

### Motivation
We had this temporarily hidden while we did some internal changes, but now it works like the others and should be documented.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

buraizu reviewed (commented):
Updates to French and Japanese documentation are generally handled through Transifex. Could you confirm if the French and Japanese updates have gone through that process already?

AdrianLC commented:
> Updates to French and Japanese documentation are generally handled through Transifex. Could you confirm if the French and Japanese updates have gone through that process already?

We had it hidden with as a comment, but they translated the comment when we added it months ago. Now I just moved it above but did not change the text. I thought this would save unnecessary translation work but open to just remove it if you think it's safer.

AdrianLC commented:
ping @buraizu
